client/landing/jetpack-cloud/sections/scan/main.tsx

```diff
@@ -205,6 +207,15 @@ class ScanPage extends Component< Props > {
 					) }
 				</p>
 				{ this.renderContactSupportButton() }
+				{ isEnabled( 'jetpack-cloud/on-demand-scan' ) && (
+					<Button
+						href={ `/scan/${ siteSlug }` }
```

Comment:
Why do we need the `href` here? It seems it makes this button being rendered as an `a` tag instead of a `button`. I believe this should be a `button` since it triggers an action and doesn't redirect the user to anywhere else.

Also, I believe it doesn't follow our style guide (maybe because it's being rendered as an `a` tag).
![image](https://user-images.githubusercontent.com/3418513/81420979-23e23200-9127-11ea-9a61-5a3f58c41c61.png)

How it should look like.
![image](https://user-images.githubusercontent.com/3418513/81420712-bd5d1400-9126-11ea-919c-747becfabc0f.png)

Reply:
You make a good point here, @rcanepa -- I hadn't thought to check the style guide. I also wasn't aware that `Button` exposes an `href` property!

I agree that we're performing an action here, not moving to a different page. In light of this, maybe we can remove the `href` value all together since `onClick` is performing as expected.

Reply:
I believe that should be the way to go, but maybe there is a reason for this `Button` to have  a `href` property that we aren't aware of. @ChaosExAnima is this normal in Calypso's codebase?

Reply:
@rcanepa - Not to my knowledge, no. I'd also say remove the `href`.